doc/manual/merging.md

```diff
@@ -256,9 +261,17 @@ final record:
 ## Merging records with metadata
 
 Metadata can be attached to values thanks to the `|` operator. Metadata
-currently includes contract annotations, default value, merge priority, and
+currently include contract annotation, default value, merge priority, and
```

Comment:
I actually find the previous formulation "Metadata currently includes contract annotations" more natural 😕 I don't have a good rationale, though.

Reply:
I don't have a good one either :sweat_smile: I think data is technically plural (the plural of _datum_) but is both plural and singular in practice. Maybe should I summon a native speaker such as @matthew-healy